package.json

```
@@ -17,10 +17,10 @@
     "build:icons": "yarn react build:icons",
     "test": "yarn react test && yarn e2e test",
     "prepare": "husky install",
-    "version:next": "yarn changeset version --snapshot next-$(git rev-parse --short=7 HEAD) && yarn build",
+    "version:next": "yarn changeset version --snapshot next-$(git rev-parse --short=7 HEAD) && yarn angular build",
     "publish:next": "yarn changeset publish --tag next",
-    "version:latest": "echo \"not implemented until stable release\"",
-    "publish:latest": "echo \"not implemented until stable release\""
+    "version:latest": "yarn changeset version && yarn angular build",
```

Actually, `&& yarn angular build` is not actually necessary, because `publish` action runs on the main branch after the new versions have been merged. So the first build (chained off `yarn install`) would have already captured the new version.
```suggestion
    "version:latest": "yarn changeset version",
```

---

With discussion on https://github.com/aws-amplify/amplify-ui/pull/782#discussion_r752555234, I'll keep this command as-is for manual deploy purpose only.

`@changesets/action` will use its native versioning logic.